miniscope_io/stream_daq.py

```diff
@@ -494,15 +555,17 @@ def capture(
         self.terminate.clear()
 
         # Queue size is hard coded
-        queue_manager = multiprocessing.Manager()
-        serial_buffer_queue = queue_manager.Queue(
+        shared_resource_manager = multiprocessing.Manager()
+        serial_buffer_queue = shared_resource_manager.Queue(
             10
         )  # b'\x00' # hand over single buffer: uart_recv() -> buffer_to_frame()
-        frame_buffer_queue = queue_manager.Queue(5)  # [b'\x00', b'\x00', b'\x00', b'\x00', b'\x00']
+        frame_buffer_queue = shared_resource_manager.Queue(5)
```

Comment:
i think these should be pretty big, right? 5 seems small

Reply:
Yes, I increased the default to 100 and also put these queue sizes in runtime variables.